docs/setup/pi4humble.md

```diff
@@ -0,0 +1,94 @@
+# Connect Create® 3 to Raspberry Pi® 4 and set up ROS 2 Humble
+
+## Before you start
+!!! attention
+    **These directions are written for someone with experience with embedded Linux and basic embedded computers.**
+It is highly recommended to read through the following documents before beginning:
+
+* [How to install Ubuntu Server on your Raspberry Pi](https://ubuntu.com/tutorials/how-to-install-ubuntu-on-your-raspberry-pi)[^1] - official Canonical documentation
+* [Installing ROS 2 on Ubuntu Linux](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html)[^1] - official Open Robotics documentation
+
+## Step-by-step
+
+1. Download and install the appropriate version of the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) on your computer.
+
+1. Open the imager and under operating system, select "Other general-purpose OS" then "Ubuntu" and finally, "Ubuntu Server 22.04 LTS (64-bit)".
+
+1. Insert your microSD card into your computer and under storage in the Raspberry Pi Imager, select your microSD card. 
+
+1. Select the gear icon to optionally customize advanced options including enabling SSH, setting a unique username and password, and configuring a wireless LAN. 
+
```

Comment:
Is there a step missing here to click the "write" button?

Reply:
Also, is there an option to tell the imager not to eject after write?

Reply:
Yup - good catch. I've updated to reflect both suggestions